changelog.json

```diff
@@ -1,6 +1,18 @@
 {
   "releases": {
-      "1.7.0-beta2": [
+    "1.7.0-beta3": [
+      "[New] Workflow that allows users to take changes with them to new branch - #7248",
```

Comment:
The PR for this is https://github.com/desktop/desktop/pull/7210. Also, should we call out stashing since taking changes to a new branch is half of the story?

Reply:
@iAmWillShepherd which issue/PR do you think is the ideal one for people to go directly to to learn about this feature? A couple of suggestions:

 - the PR implementing this work - #7210 (which is a long discussion)
 - the tracking issue that links to the relevant features - #6987 (which is still open but that's okay)
 - something else?

>  Also, should we call out stashing since taking changes to a new branch is half of the story?

Feel free to suggest changes - you're probably better placed than I am to describe this area 😁

Reply:
I made a suggestion for this in a forthcoming review 😄

Reply:
@billygriffin I'm going to take in this suggestion but @iAmWillShepherd feel free to follow up

Reply:
> @billygriffin I'm going to take in this suggestion but @iAmWillShepherd feel free to follow up

I think the original issue, https://github.com/desktop/desktop/issues/6107,  is best 👍

Reply:
> I think the original issue, #6107, is best 👍

@billygriffin do you agree?